src/ui/MessageInput/index.jsx

```diff
@@ -397,7 +397,6 @@ const MessageInput = React.forwardRef((props, ref) => {
     setHeight,
   });
 
-  const textField = ref?.current;
```

Comment:
Realized that saving the ref value in this way, it makes unexpected behavior e.g. the text value is not properly updated in realtime. So I removed this to make it the outdated value can't re-used.